pokebattle/settings/base.py

```diff
@@ -133,6 +133,20 @@ def base_dir_join(*args):
 )
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = config('SOCIAL_AUTH_GOOGLE_OAUTH2_KEY', default='')
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = config('SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET', default='')
+SOCIAL_AUTH_FIELDS_STORED_IN_SESSION = ['invite_key',]
+SOCIAL_AUTH_PIPELINE = (
+    'social_core.pipeline.social_auth.social_details',
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.user.get_username',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+    'users.auth_pipeline.validate_invite_key',
+    'users.auth_pipeline.create_invite_battle',
+)
+SOCIAL_AUTH_NEW_USER_REDIRECT_URL = '/invite/validate'
```

Comment:
This could be an issue with new users that were not invited by anyone. They'd be redirected to the `invite/validate/` page, but since they have no invites, it'll raise `DoesNotExist: Invite matching query does not exist.` (500 error).

Reply:
Also, you can use the reverse syntax for urls here, so `/invite/validate` can be replaced with `auth:validate-invite`.